cmd/signature-v4.go

```diff
@@ -162,7 +162,7 @@ func doesPolicySignatureMatch(formValues http.Header) APIErrorCode {
 func compareSignatureV4(sig1, sig2 string) bool {
 	// The CTC using []byte(str) works because the hex encoding
 	// is unique for a sequence of bytes. See also compareSignatureV2.
-	return subtle.ConstantTimeCompare([]byte(sig1), []byte(sig2)) == 1
+	return true //subtle.ConstantTimeCompare([]byte(sig1), []byte(sig2)) == 1
```

Comment:
can we add a comment on the doc function that this function is disabled for fulfilling Storj requirements?

Reply:
Comments added.